server/testutils/factories/index.ts

```diff
@@ -1,6 +1,22 @@
-import courseFactory from './course'
+import type { Factory } from 'fishery'
+
+import courseFactory from './course' // eslint-disable-line import/no-cycle
+import courseAudienceFactory from './courseAudience'
 import courseOfferingFactory from './courseOffering'
 import coursePrerequisiteFactory from './coursePrerequisite'
 import organisationFactory from './organisation'
 
-export { courseFactory, courseOfferingFactory, coursePrerequisiteFactory, organisationFactory }
+const buildListBetween = <T>(factory: Factory<T>, options: { min?: number; max: number }): Array<T> => {
+  const quantity = Math.floor(Math.random() * options.max) + (options.min || 0)
+
+  return factory.buildList(quantity)
+}
```

Comment:
If we want to keep this, we need to decide where to put it to avoid the cyclical dependency

Reply:
could it live in a `factoryHelpers.ts` file maybe?